docs/release-notes/8.3.asciidoc

```diff
@@ -0,0 +1,82 @@
+* Allows highlighted fields to be investigated in Timeline ({pull}131255[#131255]).
+* Adds support to Osquery for multi-line queries ({pull}131224[#131224]).
+* Enforces the `All` global privilege setting on Osquery ({pull}130523[#130523]).
+* Adds the *Run Osquery* option to the *More actions* menu (*...*) in the Alerts table ({pull}131790[#131790]).
+* Improves the performance of these actions on the bulk rule actions endpoint ({pull}130924[#130924]).
+** `add_tags`
+** `delete_tags`
+** `set_tags`
+** `add_index`
+** `delete_index`
+** `set_index_patterns`
+** `set_timeline_template`
+* Adds the *User risk* tab to the user details page ({pull}130256[#130256]).
+* Fixes a bug that caused the rule details page to crash when users opened a deleted or non-existent rule ({pull}133867[#133867]).
+* Allows threshold alerts to be investigated in Timeline if filters are not provided ({pull}133733[#133733]).
+* Prevents events from being added to cases from Timeline ({pull}133410[#133410]).
+* Fixes All Users and All Hosts page should reset the current page when sorting ({pull}133111[#133111]).
+* Removes the filter and investigate in Timeline options from the Elastic Agent status in Highlighted fields ({pull}132829[#132829], {pull}132586[#132586]).
+* Improves the copy of Timeline tooltips ({pull}132756[#132756]).
+* Fixes a validation bug that occurred when users were building a rule exception and changed the exception statement’s operator ({pull}131989[#131989]).
+* Adds a checkmark to the pagination selection on the *Exceptions lists* page ({pull}131979[#131979]).
+* Re-adds the success message that displays when users export an exceptions list ({pull}131952[#131952]).
+* Updates import toast logic ({pull}131873[#131873]).
+* Ensures an error is not generated when the `agent.version` provided by an alert is in an unexpected format ({pull}131272[#131272]).
+* Improves error checks for threshold rules ({pull}131088[#131088]).
+* Expands support for migrating legacy rule actions ({pull}130511[#130511]).
+* Fixes a bug that caused the *Add Rule Exception* flyout to unexpectedly close when users create the first exception for the rule from an alert ({pull}130187[#130187]).
+* Corrects Rule name sorting so detection rules are ordered alphabetically, regardless of their casing ({pull}130105[#130105]).
+* Improves the *Reporter* column in the Cases table ({pull}132200[#132200]).
+* Adds the option to create a new case to the Select case pane ({pull}128882[#128882]).
+* Allows {kibana-ref}/pre-configured-connectors.html[preconfigured connectors] to be used with cases ({pull}130372[#130372]).
+* Inserts the deprecated icon next to deprecated preconfigured connectors ({pull}132237[#132237]).
+* Updates the Case table so that all tags assigned to the case are displayed when users go to the case and hover over the *Tags* column ({pull}132023[#132023).
```

Comment:
```suggestion
* Updates the Case table so that all tags assigned to a case are displayed when users go to the case and hover over the *Tags* column ({pull}132023[#132023).
```